api/pyproject.toml

```diff
@@ -46,6 +46,7 @@ coverage = "7.5.4"
 django-silk = "5.3.2"
 docker = "7.1.0"
 freezegun = "1.5.1"
+marshmallow = ">=3.15.0,<4.0.0"
```

Comment:
Why is this required? I don't think this is an API dependency.

Reply:
This is because Marshmallow has recently updated to 4.0 and Safety is not compatible with this version so at least for now we need to set the version to lower than 4.0